src/hotspot/share/cds/archiveBuilder.cpp

```diff
@@ -643,6 +634,14 @@ void ArchiveBuilder::make_shallow_copy(DumpRegion *dump_region, SourceObjInfo* s
   newtop = dump_region->top();
 
   memcpy(dest, src, bytes);
+
+  // Update the hash of buffered sorted symbols for static dump
```

Comment:
Please append to the comments with ` so that the symbols have deterministic contents`